### PR TITLE
Update Installation guide

### PR DESCRIPTION
**Pull request by GerbrichFerdinands**
I hope this makes the installation process of python and ASReview a bit more smooth!
The main things I propose to change:

- [x] replace installing Anaconda by installing Python (for windows)
- [x] replace installing Anaconda by installing Miniconda (for macOS)
- [x] add screenshots on Python installation for windows users 
- [x] split the installation guide into two: one for windows and one for macos/unix users
- [x] add instruction on how to launch ASReview at the end of the installation instruction 
- [x] add an extra error to the troubleshoot section
- [x] add screenshots on Python installation for macos users

FYI: On GitHub it looks like some links are broken but I can't seem to get them right, maybe this is only related to GitHub and will be gone once this document is on rtd?

### CODE REVIEW

**Comment by J535D165** [comment]
Thanks for the PR. Very welcome.

>FYI: On GitHub it looks like some links are broken but I can't seem to get them right, maybe this is only related to GitHub and will be gone once this document is on rtd?

Extensions on Github end with `.rst`, in Sphinx, this will be `.html`. However, links are also broken in the PDF. So the following is preferred: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations.

File: docs/source/installation.rst [hunk 1]
```diff
@@ -1,88 +1,143 @@
...
-----------------
+.. code::
 
-If your Python version is 3.6+, you can start installing the ASReview
-software on your computer. 
+    pip install asreview
+
+If ASReview was installed succesfully, this command should return a bunch of text of which the last line is something like:
+
+.. code::
```
Comment (J535D165) [inline docs/source/installation.rst:62]: Missing whitespace

File: docs/source/installation.rst [hunk 2]
```diff
@@ -1,88 +1,143 @@
...
+**IMPORTANT: If you run into any issues during the installation, please consult `the troubleshooting section <installation.html#troubleshooting>`__ .**
 
-Windows
-~~~~~~~
+Install ASReview for Windows users
+----------------------------------
 
-First check if Python is already installed on your computer. Open the program
-`CMD.exe` on your computer. Type the following
+1. Install Python
```
Comment (J535D165) [inline docs/source/installation.rst:16]: Sections should be unique. `1. Install Python (Windows)`

**Comment by GerbrichFerdinands** [comment]
@J535D165 I want my links to open in a new window, but for my quicktour link for example, adding target = "_blank" doesn't make the hyperlink open in a new window 👎 

```
Please follow the steps in our |quicktour| on how to carry out your first Automated Systematic Review!

.. |quicktour| raw:: html

    <a href="https://asreview.readthedocs.io/en/latest/quicktour.html" target="_blank">Quick Tour </a>
```

But I expect this problem to fix itself once it gets to `.html`?

**Pull request closed without merging**